researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/task/tracked/TrackedDataSelectedItemsInterface.java

```diff
@@ -0,0 +1,17 @@
+package org.sagebionetworks.bridge.researchstack.task.tracked;
+
+import org.researchstack.backbone.result.StepResult;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 3/23/17.
+ */
+
+public interface TrackedDataSelectedItemsInterface {
```

Comment:
Would it make sense to call this "TrackedDataStep" or something? Remove "Interface" from the name. Remove the concept of "selected", since this could work for any collection of TrackedDataObjects, and in this case, the caller gets the "selected" item results, because they pass in the "selected" objects